site/src/components/LicenseCard/LicenseCard.tsx

```diff
@@ -1,7 +1,7 @@
-import Box from "@material-ui/core/Box"
-import Button from "@material-ui/core/Button"
-import Paper from "@material-ui/core/Paper"
-import { makeStyles } from "@material-ui/core/styles"
+import Box from "@mui/material/Box"
+import Button from "@mui/material/Button"
+import Paper from "@mui/material/Paper"
+import { makeStyles } from "@mui/styles"
```

Comment:
On ln 34 of this file, can we remove `variant="outlined"` since that's now the default variant?